tools/dump_tool/ramdumpParser.py

```diff
@@ -498,6 +504,63 @@ def unwind_backtrace_with_framepointer(self, fp, sp, lr, pc, ss):
 		self.stacksize = ss
 		self.find_stackframe_using_framepointer(frame)
 
+# Function to Parse the i/p log file in case of app crashes (which contains stackdump during assert)
+def parse_file(log_file):
+    pc_value=0
+	# Parse the contents based on tokens in log file.
+    with open(log_file) as searchfile:
+        for line in searchfile:
+            if 'up_registerdump:' in line:
+                word = line.split(':')
+
+                # word[1] contains the register name i.e R0 or R8
+                reg = word[1].split()
+
+                if reg[0] == 'R8':
+                    t = word[2].split( )
+
+                    # Check for corruption of PC value in logs
+                    if (len(word[2].split( )) != 8):
+                        print('\nAssert logs are corrupted, and we are not able to determine the value of PC.\n')
+                        continue
+
+                    # Last subword of word[2] contains the PC value
+                    pc_value = int(t[-1],16)
+                    #print('PC value : ', hex(pc_value))
+                    continue
+            # Read the app text address and size
+            if 'elf_show_all_bin_addr:' in line:
+                if '[micom]' in line:
+                    word = line.split(':')
+                    t = word[2].split(',')
+                    g_stext_app_micom = int(t[0], 16)
```

Comment:
The name of app can be changed by user.
And we can consider the common binary also. 
So I think it is not good to fix the variable name as app name.
How about like this? `g_stext_app1`, `g_stext_app2`?